.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think this should be committed. Other than that LGTM

Reply:
I was annoyed by this too, but after the Melos VSCode extension generated it the 100th time, I googled and [found this](https://github.com/blaugold/melos-code/issues/8).

But if you still say no, then I'll remove it.

Reply:
Removed.